examples/semi_supervised/plot_label_propagation_versus_svm_iris.py

```diff
@@ -66,14 +66,14 @@
 
     # Put the result into a color plot
     Z = Z.reshape(xx.shape)
-    plt.contourf(xx, yy, Z, cmap=plt.cm.Paired)
+    plt.contourf(xx, yy, Z, cmap='tab20')
```

Comment:
#8935 
i just found this line to give error with matplotlib 1.5 and have fixed this. rest seems fine.